pkg/utils/utils.go

```diff
@@ -124,3 +127,7 @@ func FormatBytes(n uint64) string {
 	}
 	return fmt.Sprintf("%.2f %siB (%d Bytes)", float64(m)/1024.0, units[i], n)
 }
+
+func SupportANSIColor(fd uintptr) bool {
+	return isatty.IsTerminal(fd) && runtime.GOOS != "windows"
```

Comment:
Threre are some Windows terminal support ANSI escape sequences.   
https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_ansi_terminals?view=powershell-7.2

> Since taking ownership of the Windows command-line in 2014, the team added several new features to the Console, including background transparency, line-based selection, support for [ANSI / Virtual Terminal sequences](https://en.wikipedia.org/wiki/ANSI_escape_code), [24-bit color](https://devblogs.microsoft.com/commandline/24-bit-color-in-the-windows-console/), a [Pseudoconsole ("ConPTY")](https://devblogs.microsoft.com/commandline/windows-command-line-introducing-the-windows-pseudo-console-conpty/), and more.

https://github.com/microsoft/terminal

Reply:
Currently displaying color support in Windows is not good and the problem has a small impact, so we have turned off the feature for now in order to solve the problem faster. Maybe in the future we will fully support color display in Windows.